src/main/java/io/jenkins/plugins/remotingopentelemetry/RemotingOpenTelemetryComputerListener.java

```diff
@@ -24,22 +41,49 @@
  */
 @Extension
 public final class RemotingOpenTelemetryComputerListener extends ComputerListener {
+    final static Logger LOGGER = Logger.getLogger(RemotingOpenTelemetryComputerListener.class.getName());
+
     @Override
     public final void onOnline(Computer c, TaskListener listener) throws IOException, InterruptedException {
         if (c instanceof MasterComputer) return;
+
         VirtualChannel vc = c.getChannel();
-        vc.call(new SyncMonitoringEngineCommand());
+        if (vc == null) return;
+
+        EngineConfiguration config = RemotingOpenTelemetryConfiguration.get().export();
+        try {
+            vc.call(new SyncMonitoringEngineCommand(config));
+        } catch (IOException | InterruptedException e) {
+            String command = SyncMonitoringEngineCommand.class.getName();
+            Node node = c.getNode();
+            String nodeName = node == null ? "unknown" : node.getNodeName();
+            LOGGER.log(Level.WARNING, "Fail to call " + command + " for " + nodeName , e);
+        }
+
         if (vc instanceof Channel) {
             Channel ch = (Channel) vc;
 
             // Send JAR files to the agent in advance to run the MonitoringEngine even in offline.
             // TODO: Enable to auto-detect the dependent JAR files.
-            ch.preloadJar(getClass().getClassLoader(), OpenTelemetrySdk.class);      // opentelemetry-sdk-<version>.jar
-            ch.preloadJar(getClass().getClassLoader(), CompletableResultCode.class); // opentelemetry-sdk-commmon-<version>.jar
-            ch.preloadJar(getClass().getClassLoader(), SdkTracerProvider.class);     // opentelemetry-sdk-trace-<version>.jar
-            ch.preloadJar(getClass().getClassLoader(), OpenTelemetry.class);         // opentelemetry-api-<version>.jar
-            ch.preloadJar(getClass().getClassLoader(), LoggingSpanExporter.class);   // opentelemetry-exporter-logging-<version>.jar
-            ch.preloadJar(getClass().getClassLoader(), Scope.class);                 // opentelemetry-context-<version>.jar
+            ch.preloadJar(getClass().getClassLoader(), OpenTelemetrySdk.class);          // opentelemetry-sdk
```

Comment:
It sounds like we will be interested in bundling these jars in the Jenkins Agent jar :-)

Reply:
I'll deal with the problem in the later PR. thanks!

Reply:
ref: https://cdeliveryfdn.slack.com/archives/C023E83AMAL/p1624026415165400?thread_ts=1624024399.162800&cid=C023E83AMAL